src/components/ENSAddress.tsx

```diff
@@ -28,18 +28,35 @@ function ENSAddressSuspended({
   )
 }
 
-export default memo<ENSAddressProps>(({ address, network }) => {
-  const { md, lg } = useBreakpoints()
-  const truncateSize = md ? (lg ? 25 : 17) : 11
-
+function ENSAddress({
+  address,
+  network,
+  truncateSize,
+}: ENSAddressProps & { truncateSize: number }) {
   return (
     <Suspense fallback={truncateMiddleIfNeeded(address, truncateSize)}>
       <ENSAddressSuspended
+        truncate
         address={address}
-        truncateSize={truncateSize}
-        truncate={!lg}
         network={network}
+        truncateSize={truncateSize}
       />
     </Suspense>
   )
+}
+
+export default memo<ENSAddressProps>(({ address, network }) => {
```

Comment:
I gave a few comments below, but we're really WET here, I think it's better to `useBreakpoints` as it were before

Reply:
Left it as in SCE